### PR TITLE
Task completed

### PR DESCRIPTION
**Pull request by NikolayFatkullin**
- Created classes: Bet, BetDaoImpl, ConsoleHandler, Main, Player, PlayerDaoImpl, Storage;

- Created interfaces: PlayerDao, BetDao;

### CODE REVIEW

**Review by ivlytvynets — APPROVED** [review]

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.BetDaoImpl;
+import core.basesyntax.model.Bet;
+import java.util.Scanner;
+
+public class ConsoleHandler {
+    BetDao betDao = new BetDaoImpl();
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java:9]: access modifiers

Comment (dmytro-hryhoruk) [reply src/main/java/core/basesyntax/controller/ConsoleHandler.java:9]: What about other Dao?

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+            Bet bet = null;
+            try {
+                String[] betData = command.split(" ");
+                int value = Integer.parseInt(betData[0]);
+                double risk = Double.parseDouble(betData[1]);
+                bet = new Bet(value, risk);
+            } catch (NumberFormatException e) {
+                System.out.println("Please input correct data");
+            }
+            betDao.add(bet);
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java:27]: let's add only if bet != nul and user getAll() to print all of them at the end

**Review by dmytro-hryhoruk — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+                int value = Integer.parseInt(betData[0]);
+                double risk = Double.parseDouble(betData[1]);
+                bet = new Bet(value, risk);
+            } catch (NumberFormatException e) {
+                System.out.println("Please input correct data");
+            }
+            if (bet != null) {
+                betDao.add(bet);
+            }
+            System.out.println(bet == null ? null : bet.toString());
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java:34]: This check and calling toString() done automatically  by System.out.println(check it)

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request merged**